package.json

```diff
@@ -99,8 +99,8 @@
     "webdriverio": "^4.9.8",
     "webpack": "4.19.0",
     "webpack-cli": "^2.0.13",
-    "webpack-merge": "^4.1.2",
-    "webpack-serve": "^1.0.2"
+    "webpack-dev-server": "^3.1.9",
```

Comment:
Is this valid with IE 10? I feel like had this locked down at one point

Reply:
It's not guaranteed. Webpack-serve didn't work for IE10 either, thats one reason why we made serve static (and recommend that in our doc for IE 10 testing)

https://github.com/webpack/webpack-dev-server#browser-support